docs/guide/docs/advanced/hosting.md

```diff
@@ -111,17 +125,12 @@ docker run -d \
 botpress/server:$TAG
 ```
 
-### Hosting Duckling and the Language Server
-
-This step is optional. Your Botpress installation will use our hosted services by default.
+Choose to either run one of two containers (two containers are recommended).
 
-Choose to either run one of two containers (two containers is recommended).
+### Running a Single Container
```

Comment:
This section seems a bit light on explanation regarding running single container. Is there really not much to say?

Reply:
Anything you should install before running a docker container?